### ISSUE TITLE
Add `cluster upgrade` command for Fluvio Cluster CLI`

### ISSUE THREAD

**Issue by sehz** [infinyon/fluvio#665]
`cluster upgrade` will upgrade existing k8 fluvio installation.   By default, it will upgrade to same version in the CLI.

Instead of invoking, `helm install`, it will perform `helm upgrade`.   It will have following argument similar to `install`:
```
-- chart-version <chart-version>
--image-version <image-version>
--namespace <namespace>
--install-name <install-name>
--chart-location <chart-location>
--skip-sys  // disable upgrading sys chart
```

By default, it will upgrade sys chart as well unless `--skip-sys` argument is present.

**Comment by sehz** [comment]
should revisit #551 when doing this

**Comment by sehz** [comment]
need to fix https://github.com/infinyon/flv-util/issues/10 as part this

**Comment by nicholastmosher** [comment]
Closed by #709
